governance/polls/Offboard USDT Collateral Types - September 27, 2021.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Offboard USDT Collateral Types - September 27, 2021
+summary: Signal your support or opposition to Offboard USDT Collateral Types.
+discussion_link: https://forum.makerdao.com/t/signal-request-offboard-usdt-collateral-types/10223
+vote_type: Plurality Voting
+categories:
+   - Collateral
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-09-27T16:00:00
+end_date: 2021-09-30T16:00:00
```

Comment:
Are we sure about this end date? The forum thread says the poll will run for 2 weeks - can we double check? @Hipogeo @prose11

Reply:
Note this was my mistake - thanks Sebix